hca/upload/cli/list_file_status_command.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+from hca.upload.cli.common import UploadCLICommand
+from hca.upload.upload_submission_state import FileStatusCheck
+
+
+class ListFileStatusCommand(UploadCLICommand):
+    """
+    Print status of file in an upload area
+    """
+    @classmethod
+    def add_parser(cls, upload_subparsers):
+        list_file_statuses_parser = upload_subparsers.add_parser('status', help=cls.__doc__,
+                                                                 description=cls.__doc__)
+        list_file_statuses_parser.set_defaults(entry_point=ListFileStatusCommand)
+        list_file_statuses_parser.add_argument('env', help="Environment the upload area was created in")
```

Comment:
Could not perhaps the env and uuid be optional arguments, and default to the env configured in `.config/hca/config.json` and the current upload area, respectively?

I imagine that the usual usage of this file will be to check the status of files you just uploaded.  In that case having to specify env and area is overkill.